events/guildCreate.js

```diff
@@ -1,8 +1,12 @@
-// This event executes when a new guild (server) is joined.
+// // This event executes when a new guild (server) is joined.
```

Comment:
Commenting out a comment?

Reply:
Accidentally done when I commented out the old events while re-writing them to the new structure. Fixed.